### PR TITLE
Filesystem monitor library

### PR DESCRIPTION
**Pull request by iivanou**

### CODE REVIEW

File: filesystem_service/src/filesystem_service/daemon.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+from logging import handlers
+import os
+import signal
+import sys
+from time import sleep
+
+from daemonize import Daemonize  # pylint: disable=import-error
+
+
+class Daemon(Daemonize):
```
Comment (AndreiPashkin) [inline filesystem_service/src/filesystem_service/daemon.py]: This is not supposed to be part of filesystem monitor.

File: filesystem_service/src/filesystem_service/handlers.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        :return:
+        """
+
+        for i_event in IMP_EVENTS:
+            if event.__dict__.get(i_event, False):
+                self.source.event(dumps(event.__dict__))
+
+
+# pylint: disable=too-few-public-methods
+class FileSystemDbusHandler(dbus.service.Object):
```
Comment (AndreiPashkin) [inline filesystem_service/src/filesystem_service/handlers.py:45]: Why DBus?

**Review by AndreiPashkin — CHANGES_REQUESTED** [review]
I would add unit tests.

File: filesystem_service/src/filesystem_service/handlers.py (hunk 2 above)
Comment (iivanou) [reply filesystem_service/src/filesystem_service/handlers.py:45]: Why not?

**Comment by iivanou** [comment]
Need to split into pieces

**Pull request closed without merging**